cli/cmd/list.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/list"
+	"github.com/tarantool/tt/cli/modules"
+)
+
+// NewListCmd creates list command.
+func NewListCmd() *cobra.Command {
+	var listCmd = &cobra.Command{
+		Use:   "list",
+		Short: "Show enabled applications.",
```

Comment:
There is no dot in the help of other commands.

Reply:
removed